deploy/iso/minikube-iso/arch/aarch64/package/containerd-bin-aarch64/containerd-bin.mk

```diff
@@ -9,10 +9,6 @@ CONTAINERD_BIN_SITE = https://github.com/containerd/containerd/releases/download
 CONTAINERD_BIN_SOURCE = containerd-$(CONTAINERD_BIN_VERSION)-linux-arm64.tar.gz
 CONTAINERD_BIN_STRIP_COMPONENTS = 0
 
-define CONTAINERD_BIN_CONFIGURE_CMDS
-	mkdir -p $(TARGET_DIR)/etc/containerd/containerd.conf.d
```

Comment:
Last time I tried to use drop-in configuration files they were not useful since drop-in file overrides the entire section. If this was fixed in containerd 2 we can create this directory. to make it easier to use drop-in configuration. This can be moved to the INSTALL_TARGET_CMDS section.